plugin-server/src/utils/db/hub.ts

```diff
@@ -172,7 +172,7 @@ export async function createHub(
     const kafka = new Kafka({
         clientId: `plugin-server-v${version}-${instanceId}`,
         brokers: serverConfig.KAFKA_HOSTS.split(','),
-        logLevel: determineNodeEnv() === NodeEnv.Test ? logLevel.ERROR : logLevel.DEBUG,
+        logLevel: KAFKAJS_LOG_LEVEL_MAPPING[serverConfig.KAFKAJS_LOG_LEVEL] ?? logLevel.WARN,
```

Comment:
We might want to support lowercase _and_ uppercase values, especially since there's no error thrown if the log level is invalid